### ISSUE TITLE
请问macbert训练的loss 检错和纠错的权重是不是写反了

### ISSUE THREAD

**Issue by Helenailse1** [shibing624/pycorrector#301]
readme 网络结构图 检错权重w ，纠错权重1-w 
代码中outputs = (det_loss,
                       bert_outputs.loss,
                       self.sigmoid(prob).squeeze(-1),
                       bert_outputs.logits)
train_step 中loss = self.w * outputs[1] + (1 - self.w) * outputs[0]

**Comment by shibing624** [comment]
w是个指定参数，0-1范围内都可以，当前取0.3，也可以是0.7，正反无所谓。
